rest-api-spec/keywords.json

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
Why do we need this file?

Reply:
Besides validating the rest-api-specs against the schema.json, the "ValidateRestSpecPlugin" also validates if any of the client keywords listed in this file are being used in rest-api-specs and makes sure that these keywords are not used in any of the rest-api-specs.
https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/precommit/ValidateRestSpecPlugin.java

Line -59 to 69